app/services/pupil_premium_importer.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class PupilPremiumImporter
+  def initialize(logger, start_year = Time.zone.now.year, source_file = "")
```

Comment:
Better to have source_file default to nil rather than "". Was there a good reason for picking ""

Reply:
None at all, will change